db/schema.rb

```diff
@@ -75,85 +75,13 @@
     t.text     "recycling",                                                :limit=>512
     t.integer  "minbudget_realcurrency"
     t.integer  "maxbudget_realcurrency"
-    t.integer  "safetybag_crewsize"
```

Comment:
Hm is removing these part of this PR? I thought there was a pass at removing a bunch of columns already in master?

Reply:
Derp, no. Something went wrong here, will fix.